libs/backend/src/cast_vote_records/export.ts

```diff
@@ -423,6 +429,76 @@ async function exportSignatureFileToUsbDrive(
   return ok();
 }
 
+/**
+ * Updates the creation timestamp of a directory and its children files (assuming no
+ * sub-directories) using the one method guaranteed to work:
+ * ```
+ * cp -r <directory-path> <directory-path>-temp
+ * rm <directory-path>
+ * mv <directory-path>-temp <directory-path>
+ * ```
+ */
+async function updateCreationTimestampOfDirectoryAndChildrenFiles(
+  directoryPath: string
+): Promise<void> {
+  const tempDirectoryPath = `${directoryPath}-temp`;
+  await fs.mkdir(tempDirectoryPath);
+  const fileNames = (await fs.readdir(directoryPath, { withFileTypes: true }))
+    .filter((entry) => entry.isFile())
```

Comment:
This line means the function silently ignores directories. If we're confident there won't be any directories, should we `throw` if we encounter one? Otherwise a change to the CVR directory structure will cause this function to slowly corrupt the CVRs.

Reply:
Okay, did decide to throw here and in one other place relevant to artifact authentication https://github.com/votingworks/vxsuite/pull/3972/commits/28f0f9002b2321df5ee839eef496754444393862

I've been worried about hidden directories (and files) created by file system tools causing unexpected problems, but ultimately, I think the rigidity makes sense here